conf/bk_server.conf

```diff
@@ -41,7 +41,8 @@
 bookiePort=3181
 
 # Configure the bookie to allow/disallow multiple ledger/index/journal directories
-# in the same filesystem disk partition
+# in the same filesystem disk partition.
+# Defaults to true.
 # allowMultipleDirsUnderSameDiskPartition=false
```

Comment:
Can you change also this line?

Reply:
If we change this value then we will change the default behaviour

@RaulGracia please also take a look. I think that in Pravega we rely on this file for default values

Reply:
@eolivelli I thought that wouldn't affect the default value since it's commented out?  Wouldn't it still default to true based on the [sources](https://github.com/apache/bookkeeper/blob/032aef7e75f6b3487a6611a2b8af8306b9a3f7bb/bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java#L3499-L3500).

I wasn't sure if the commented out examples were supposed to match the default values, or were to there to provide an each way to uncomment and override the default value.  I think there are examples of both cases currently in the config file.  Setting it to match the default value as @nicoloboschi suggested does seem a bit more intuitive to me.

Reply:
@nicoloboschi I updated the example config to match the default `true`